.appveyor.yml

```diff
@@ -42,8 +42,8 @@ install:
 
 before_build:
   # setup env
-  - CALL "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Auxiliary\Build\vcvars64.bat"
-  - SET PATH=%PATH%;C:\Qt\5.15.2\msvc2019_64\bin;%CACHE_DIR%\jom
+  - CALL "C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Auxiliary\Build\vcvars64.bat"
+  - SET PATH=%PATH%;C:\Qt\5.15.2\msvc2022_64\bin;%CACHE_DIR%\jom
```

Comment:
You still have to use Qt binaries built with MSVC2019.

Reply:
Seems you are right looking at their docs, they used `C:\Qt\5.15.2\msvc2019_64` for both `Visual Studio 2019/2022 Images`, waiting back from them to see if this will be changed in a future release......if not, will change to `C:\Qt\5.15.2\msvc2019_64`

Reply:
@glassez changed back to `C:\Qt\5.15.2\msvc2019_64` as `Qt binaries` built with `MSVC2022` aren't available from `Qt` themselves yet but `Appveyor` do plan to use them when they are available.

**Appveyor now passing.**
____

Quick question just for my own curiosity, why do we define the cores to be used -> `jom -j2 -f Makefile.Release` instead of letting `jom` auto-detect e.g. `jom -f Makefile.Release`?

Reply:
>Quick question just for my own curiosity, why do we define the cores to be used -> jom -j2 -f Makefile.Release instead of letting jom auto-detect e.g. jom -f Makefile.Release?

Don't know.